examples/swift/hello_world/ViewController.swift

```diff
@@ -59,14 +60,22 @@ final class ViewController: UITableViewController {
       .newStreamPrototype()
       .setOnResponseHeaders { [weak self] headers, _ in
         let statusCode = headers.httpStatus ?? -1
-        var message = "received headers with status \(statusCode)"
+        let message = "received headers with status \(statusCode)"
+
+        var headerMessage = ""
+        headers.allHeaders().forEach { name, value in
+          if kFilteredHeaders.contains(name) {
+            headerMessage += "\(name): \(value.joined(separator: ", "))\n"
+          }
+        }
```

Comment:
```suggestion
        let headerMessage = headers.allHeaders()
          .filter { kFilteredHeaders.contains($0.key) }
          .map { "\($0.key): \($0.value.joined(separator: ", "))" }
          .joined(separator: "\n")
```

Reply:
This keeps the value immutable and is a bit more idiomatic

Reply:
Nice, thanks.